### PR TITLE
Fixed syntax error, added GovCloud mapping

### PR DESCRIPTION
**Pull request by trend-scottb**

### CODE REVIEW

File: templates/marketplace/dsm-mp.template [hunk 1]
```diff
@@ -251,6 +251,8 @@ Mappings:
     sa-east-1:
       PerHost: ami-1f275373
       BYOL: ami-3d057151
+    us-gov-west-1:
+      BYOL: ami-f5de5e94
```
Comment (santiagocardenas) [inline templates/marketplace/dsm-mp.template]: Missing the PerHost AMI?

Comment (trend-scottb) [inline templates/marketplace/dsm-mp.template]: BYOL is only supported right now in GovCloud

Comment (santiagocardenas) [inline templates/marketplace/dsm-mp.template]: Is this prevented via a rule/condition and/or documented in the deployment guide?

Comment (trend-scottb) [inline templates/marketplace/dsm-mp.template]: Both, it will fail the check in per host template and it's documented in the deployment guide.

**Comment by santiagocardenas** [comment]
Please let me know when this is tested out and I'll merge.

**Comment by trend-scottb** [comment]
Testing is complete, let me know when it's live and I can sanity test the live s3 bucket

**Pull request merged**